rustbus/fuzz/fuzz_targets/fuzz_unmarshal.rs

```diff
@@ -4,21 +4,21 @@ extern crate libfuzzer_sys;
 extern crate rustbus;
 
 fuzz_target!(|data: &[u8]| {
-    let (hdrbytes, header) = match rustbus::wire::unmarshal::unmarshal_header(data, 0) {
+    let mut cursor = rustbus::wire::unmarshal_context::Cursor::new(data);
+    let header = match rustbus::wire::unmarshal::unmarshal_header(&mut cursor) {
+        Ok(head) => head,
+        Err(_) => return,
+    };
```

Comment:
`let Ok(header) = ... else { return };` may work nicely here.

Reply:
True. I'll keep an I open if this pattern is somewhere else 👍